scss/atoms/typography/_mixins.scss

```diff
@@ -0,0 +1,26 @@
+// ==========================================================================
```

Comment:
@MichaelArestad well its not necessarily multi line it the only valid style in normal css. I personally don't think its good practice to use different comment styles in different files so we should stick to valid css comments i don't think using `//` has any benefit over `/* */` other then not typing 2 chars which git stripped out in real build anyway i think clarity and consistency should prevail here. @sfrisk what do you think?